solutions/usaco-786.mdx

```diff
@@ -240,4 +240,52 @@ public class lifeguards {
 ```
 
 </JavaSection>
+
+<PySection>
+
+```python
+import sys
+
+sys.stdin = open("lifeguards.in", "r")
+sys.stdout = open("lifeguards.out", "w")
+
+t = sys.stdin.readline
+
+n = int(t())
+
+if n == 1:
+	sys.stdout.write(str(0))
+	exit()
+
+sweep = []
+for _ in range(n):
+	a, b = map(int, t().split())
+	sweep.append([a, b])
+
+sweep.sort(key=lambda x:x[0])
```

Comment:
```suggestion
sweep.sort(key=lambda x:x[0])
```
sorting in python automatically sorts by first element?

Reply:
yeah